### PR TITLE
Fix logging

### PR DESCRIPTION
**Pull request by ivarconr**
This allows the users of 'unleash-server' to programatically change the logger implementation. #175

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -37,4 +37,37 @@ Available unleash options includes:
...
+### What is a logger provider?
+ 
+A logger provider is a function which takes the name of a logger and returns a logger implementation. For instance, the following code snippet shows how a logger provider for the global `console` object could be written:
+ 
+```javascript
+function consoleLoggerProvider (name) {
+  // do something with the name
+  return {
+    debug: console.debug.bind(console),
+    info: console.info.bind(console),
```
Comment (SimenB) [inline docs/getting-started.md]: Is binding console necessary in newer nodes?

File: docs/getting-started.md [hunk 2]
```diff
@@ -37,4 +37,37 @@ Available unleash options includes:
...
+ 
+Custom logger providers need to be set *before requiring the `unleash-server` module*. The following example shows how this can be done:
+ 
+```javascript
+// first configure the logger provider
+const unleashLogger = require('unleash-server/logger');
+unleashLogger.setLoggerProvider(consoleLoggerProvider);
+ 
+// then require unleash-server and continue as normal
+const unleash = require('unleash-server');
```
Comment (SimenB) [inline docs/getting-started.md]: Missing closing backticks?

File: docs/getting-started.md (hunk 1 above)
Comment (ivarconr) [inline docs/getting-started.md]: no

**Comment by sveisvei** [comment]
👍

**Pull request merged**